nixopsaws/resources/route53_hosted_zone.py

```diff
@@ -31,7 +31,8 @@ def __init__(self, xml, config):
         self.private_zone = config["privateZone"]
         self.zone_name = config["name"]
         self.associated_vpcs = config["associatedVPCs"]
-        map(lambda x: x.pop("_module"), self.associated_vpcs)
+        for vpc in self.associated_vpcs:
+            vpc.pop("_module")
```

Comment:
This is the "tricky case" mentioned in https://docs.python.org/3.0/whatsnew/3.0.html#views-and-iterators-instead-of-lists where we don't actually care about the list coming back. Implemented the swap to a for loop, as upstream recommends.